### PR TITLE
Add exceptions to drinks so you can slap them in the microwave

### LINKED ISSUE #3630: Transfering flour to microwave just got very fucky
Floursacks won't transfer flour to microwave, milk cartons won't transfer flour to microwave. Kitchen got a large beaker but you can't set the transfer amount to five and some recipes require you to go five by five units. Small beakers can be set to transfer five units but you gotta ask chemist or cargo for that so good luck getting it. Please fix.

### PR DESCRIPTION
**Pull request by ghost**
Microwaves, being !!FUN!! in general, can't accept reagents anymore from "drinks" (which include milk bottles and flour bags) due to recent changes. Also no, I don't have any idea about large beakers not having an option to transfer five units

Should fix #3630

### CODE REVIEW

**Comment by ComicIronic** [comment]
Why not make microwaves an open container?

**Comment by Probe1** [comment]
Beakers only transferring 10u is good that way.  You can use a syringe to draw out 5u, or fill up both condiment bottles or flour sacks if you want to deal out 5u dosages.  

Having the option to add 10 flour in one go is very useful for a chef.  Things like pies require 10u.  It can really help a chef improve their speed.

**Comment by ComicIronic** [comment]
While having the option to transfer 10 units is good, ONLY having the option to transfer 10 units is arguably not.

**Comment by dylanstrategie** [comment]
Alright now it's working correctly

**Comment by Probe1** [comment]
@ComicIronic that's why I wrote the three primary ways around it.  You'll have three flour bags and if god forbid someones an idiot and throws them out when they're empty, you can also order more or alternatively use the condiment maker to receive the same thing but in less special looking sprites.

**Comment by dylanstrategie** [comment]
Need a decision on this

**Comment by 9600bauds** [comment]
@probe1 I'd think the time you save by clicking once rather than twice to make an apple pie is more than eclipsed by the time you spend juggling around a syringe when you want to make all the recipes that require 5 or 15 flour. Ideally beakers would have the option for both 5 and 10 units - why not?

File: code/modules/reagents/reagent_containers/food/drinks.dm [hunk 1]
```diff
@@ -196,7 +196,7 @@
 
 	else //Something like a glass. Player probably wants to transfer TO it.
 
-		if(!istype(target, /obj/item/weapon/reagent_containers)) //Not a reagent container, we most likely shouldn't be dealing with it here !
+		if(!istype(target, /obj/item/weapon/reagent_containers) && !istype(target, /obj/machinery/microwave)) //Not a reagent container or microwave, we most likely shouldn't be dealing with it here !
```
Comment (d3athrow) [inline code/modules/reagents/reagent_containers/food/drinks.dm]: I suggest refactoring this into a proc that's called on the target to return if we should try to interact with it or just slap it/splash it.

**Comment by Probe1** [comment]
shrug.  because it fucks with chemists, scientists, mechanics and everyone else who just want to mix reagents or add acid to their printers.  unless i'm power-cheffing out pies I won't use a beaker no big deal

**Pull request merged**